.buildkite/pipeline.yml

```
@@ -72,11 +72,28 @@ steps:
           .buildkite/commands/prepare-environment.sh
 
           .buildkite/commands/install-node-dependencies.sh
+
           node ./scripts/prepare-dev-build-version.mjs
 
           echo "--- :node: Building Binary"
           npm run make:macos-{{matrix}}
 
+          # Local trial and error show this needs to run before the DMG generation (obviously) but after the binary has been built
+          echo "--- :hammer: Rebuild fs-attr if necessary before genrationg DMG"
+          case {{matrix}} in
+            universal | x64)
+              echo "Rebuilding fs-xattr for {{matrix}} architecture"
+              npm rebuild fs-xattr --verbose --cpu universal
```

We probably don't need the logs from this command.

```suggestion
              npm rebuild fs-xattr --cpu universal
```